docs/src/components/ButtonGroup/ButtonGroup.stories.mdx

```diff
@@ -29,7 +29,7 @@ import {ButtonGroup} from 'dooboo-ui';
 | data                   | `string[]`                 | List of strings to be rendered in each button. |
 | color?                 | `string`                   | Represents main color.                         |
 | onPress?               | `(i: number) => void`      | `callback` for press event.                    |
-| initialIndex?          | `number`                   | Initial `index` to be selected.                |
+| selected?              | `number`                   | selected `index`.                              |
```

Comment:
It feels like `selected` props should be boolean. How about renaming it to `selectedIndex`?

Reply:
Yes. you are definitely right!